src/app/myworkflows/my-workflow/my-workflow.component.html

```diff
@@ -129,6 +138,21 @@ <h3 class="sidebar-header">Shared with me</h3>
                 >
                   Register {{ entryType$ | async | titlecase }}
                 </button>
+
+                <li *ngIf="entryType === EntryType.BioWorkflow">
+                  If there are already workflows on Dockstore added by other users that you should have access to, press the button below.
+                </li>
+                <button
+                  *ngIf="entryType === EntryType.BioWorkflow"
+                  mat-raised-button
+                  color="primary"
+                  type="button"
+                  (click)="addToExistingWorkflows()"
+                  [disabled]="isRefreshing$ | async"
+                  matTooltip="Add yourself to existing Dockstore workflows that you should have access to on remote sites."
```

Comment:
Suggest removing "should".

Instead of "remote sites", suggest "source control sites".